src/common/core/coretype.js

```diff
@@ -1102,16 +1144,7 @@ define([
         };
 
         this.processRelidReservation = function (node, relid) {
```

Comment:
I kept the one defined which is easier to use for most cases (a new relid was generated - process the bases..). The one taking a number is really only used in `setBase`.